tests/openthread/README.md

```diff
@@ -0,0 +1,34 @@
+## OpenThread on RIOT
+
+This test demonstrates how to use NEST's OpenThread stack from [OpenThread](https://github.com/openthread/openthread)
+from RIOT. It shows how to use OpenThread UDP capacilities and how to interact with the API.
+
+## Quick usage
+
+To test OpenThread on RIOT, you can do the following:
+
+1. Flash some nodes with `make BOARD=<target> clean all flash`
+2. Set the PAN ID in all nodes to the same with ifconfig command (e.g `ifconfig set panid 0x1234`).
```

Comment:
Found the cli list on [their github](https://github.com/openthread/openthread/tree/master/src/cli) but if you have better ones, let's go for them.